options/locale/locale_en-US.ini

```diff
@@ -1086,6 +1086,7 @@ pulls.merge_conflict = Merge Failed: There was a conflict whilst merging: %[1]s<
 pulls.rebase_conflict = Merge Failed: There was a conflict whilst rebasing commit: %[1]s<br>%[2]s<br>%[3]s<br>Hint:Try a different strategy
 pulls.unrelated_histories = Merge Failed: The merge head and base do not share a common history. Hint: Try a different strategy
 pulls.merge_out_of_date = Merge Failed: Whilst generating the merge, the base was updated. Hint: Try again.
+pulls.push_rejected = Merge Failed: The push was rejected with the following message:<br>%s
```

Comment:
Looks OK, but why adding `<br>`?

Reply:
It forces the message on to a new line. Probably we should be putting these into a `<p>` but IIRC that breaks the rendering